packages/utilities/exposure-scanning/src/index.js

```diff
@@ -0,0 +1,47 @@
+import parseArgs from '../args/index';
+import { fetchPr, fetchIssue } from '../fetch/index';
+import { scanPr, scanIssue } from '../scan/index';
+import { patchPr, patchIssue } from '../patch/index';
+
+const redactPr = async (reqBody, regex) => {
+  const pr = await fetchPr(reqBody);
+
+  const scannedPr = await scanPr(pr, regex);
+
+  if (scannedPr.foundMatch) {
+    await patchPr(reqBody, scannedPr);
+    throw new Error('Match found.');
```

Comment:
I'm not familiar with throw and it's usage so bare with. 

So when the PR/Issue has been patched after a match was found, does the throw just exit the code here and return the error string `Match found`, which in turn means it won't reach line 44 where it returns `No matches found.`? If so shall it also include that it has been patched alongside finding a match in the error string?

Reply:
Yes that's correct 👍 And yeah good idea regarding the error message, I'll make that change now